packaging/nnfw-plugin.pc.in

```diff
@@ -1,5 +1,5 @@
 Name: nnfw-plugin
```

Comment:
Maybe you should change the name of this file.

Reply:
I've updated `rules` to update name after installation.